source/javascripts/components/StepDrawer/StepDrawer.types.ts

```diff
@@ -0,0 +1,17 @@
+export type FilterFormValues = {
+  search: string;
+  projectType: string;
+  categories: string[];
+};
+
+export type Step = {
```

Comment:
I'm just wondering how to differentiate this from the `Step` type that comes from the YML. Wouldn't it be better to call it `AlgoliaStep` instead?

Reply:
It's not even in the shape of an Algolia data. It's solely the data that the StepCard needs for rendering.